src/components/ctf-components/ctf-page/ctf-page.tsx

```diff
@@ -27,64 +32,65 @@ export default async function CtfPage() {
     metaTags.no_follow === true ? 'nofollow' : undefined,
   ].filter((x): x is string => x !== undefined);
 
+  const contentfulData = {
+     sys: { id: 'VL0GACJpBfmSRvACLvMdP'},
```

Comment:
@Polleke007  what is this? why is this needed?

Reply:
The id of the content that should be retrieved

Reply:
@Polleke007  But why is this hardcoded? This should be configured in my 'AmexioPage' (see getAmexioPage method).
It should not be done in this way